plugins/catalog-backend-module-fmeflow/src/FmeFlowEntitiesProcessor.ts

```diff
@@ -69,16 +73,40 @@ export class FMEFlowEntitiesProcessor implements CatalogProcessor {
       }
     }
 
-    // Apply ownership relations
-    doEmit(
-      entity.spec.owner,
-      {
-        defaultKind: 'FMEWorkspace',
-        defaultNamespace: selfRef.namespace,
-      },
-      RELATION_OWNED_BY,
-      RELATION_OWNER_OF,
-    );
+    // 1. Table dependency relations
+    const seenTables = new Set<string>();
+    if (Array.isArray(entity.spec.tables)) {
+      for (const { schema, table } of entity.spec.tables as Array<{ schema: string; table: string }>) {
```

Comment:
Add `tables` to `FMEWorkspaceEntity`, and use typenarrowing using `isFMEWorkspaceEntity`, instead of `isArray` and type assertion.